packages/typescript-estree/tests/lib/parse.test.ts

```diff
@@ -362,11 +361,27 @@ describe('parseAndGenerateServices', () => {
               ...config,
               extraFileExtensions,
               filePath: join(PROJECT_DIR, filePath),
+              project: './tsconfig.json',
             });
           } catch (error) {
             alignErrorPath(error as Error);
           }
         };
+      const testExtraFileExtensions =
+        (filePath: string, extraFileExtensions: string[] = ['.vue']) =>
```

Comment:
[Testing] I think this got missed in the refactors?

```suggestion
        (filePath: string, extraFileExtensions: string[]) =>
```

I can just apply it myself, about to merge.